tasks/README.md

```diff
@@ -0,0 +1,138 @@
+# Hyperdrive Deploy w/ Hardhat
+
+[Hardhat tasks](https://hardhat.org/hardhat-runner/docs/guides/tasks) are used to deploy the Hyperdrive contracts for the networks specified in [hardhat.config.ts](../hardhat.config.ts).
+
+| :warning: Warning                                                                                                         |
+|:--------------------------------------------------------------------------------------------------------------------------|
+| Redeploying the same contracts for a network will overwrite the values in the [deployments folder](../deployments/).      |
+
+## TLDR
+
+```sh
+npx hardhat deploy:all
+```
+
+## Tasks
+
+The complete list of tasks can be seen by running `npx hardhat --help` in your terminal.
+
+| :warning: Warning                                                                                                                     |
+|:--------------------------------------------------------------------------------------------------------------------------------------|
+| The bare `deploy` task is unrelated to our deployment process and SHOULD NOT BE RUN. It will be disabled/overridden in the future     |
+
+```sh
+  deploy:all                 	deploys the HyperdriveFactory and all deployer coordinators
+  deploy:coordinators:all    	deploys all deployment coordinators
+  deploy:coordinators:erc4626	deploys the ERC4626 deployment coordinator
+  deploy:coordinators:reth   	deploys the RETH deployment coordinator
+  deploy:coordinators:steth  	deploys the STETH deployment coordinator
+  deploy:coordinators:ezeth   deploys the EzETH deployment coordinator
+  deploy:factory             	deploys the hyperdrive factory to the configured chain
+  deploy:forwarder           	deploys the ERC20ForwarderFactory to the configured chain
+  deploy:instances:all       	deploys the ERC4626 deployment coordinator
+  deploy:instances:erc4626   	deploys the ERC4626 deployment coordinator
```

Comment:
Are these comments wrong? I was expected to see something like: `deploys an ERC4626 pool`